package.json

```diff
@@ -14,19 +14,20 @@
   },
   "devDependencies": {
     "@types/marked": "4.0.1",
-    "@types/node": "^18.7.16",
+    "@types/node": "18.11.9",
     "@typescript-eslint/eslint-plugin": "5.43.0",
     "@typescript-eslint/parser": "5.43.0",
     "eslint": "8.27.0",
     "eslint-config-preact": "1.3.0",
-    "wmr": "3.8.0",
-    "typescript": "4.9.3"
+    "typescript": "4.9.3",
+    "wmr": "3.8.0"
   },
   "dependencies": {
     "hoofd": "^1.2.2",
-    "marked": "4.0.10",
-    "preact": "^10.11.0",
-    "preact-iso": "2.3.0"
+    "jsdom": "^20.0.1",
+    "marked": "4.1.0",
+    "preact": "10.11.0",
```

Comment:
preact is broken since 10.11.1 due to SignalLike changes...
I can't think of a good way to do it, so downgrade once and use it
![image](https://user-images.githubusercontent.com/50351271/202815242-3dd14dbc-5a61-4bb1-a0f7-a9d6f1efa69b.png)

https://github.com/preactjs/preact/pull/3747